addons/mail/static/src/owl/store/messaging_actions.js

```diff
@@ -1331,6 +1459,37 @@ const actions = {
             userId: userIds.length ? userIds[0] : null,
         });
     },
+    /**
+     * @private
+     * @param {Object} param0
+     * @param {function} param0.dispatch
+     * @param {Object} param0.getters
+     * @param {Object} param0.state
+     * @param chatterLocalId
+     */
+    async _computeChatter({ dispatch, getters, state }, chatterLocalId){
```

Comment:
whitespace between `)` and `{`
_(would be nice having a linter for that :p)_

Reply:
I have one but I cannot launch it on messaging_actions as there are some "non-binary" conventions... And if I do I get something like follows which makes quite difficult to find a warning I need to actually fix...

>    141:28  warning  Operator '=' must be spaced                                   space-infix-ops
   144:24  warning  Operator '=' must be spaced                                   space-infix-ops
   147:13  warning  Identifier 'res_id' is not in camel case                      camelcase
   148:13  warning  Identifier 'res_model' is not in camel case                   camelcase
   150:27  warning  Operator '=' must be spaced                                   space-infix-ops
   230:23  warning  Operator '=' must be spaced                                   space-infix-ops
   577:21  warning  Operator '=' must be spaced                                   space-infix-ops
   577:29  warning  Operator '=' must be spaced                                   space-infix-ops
   608:23  warning  Operator '=' must be spaced                                   space-infix-ops
   608:28  warning  Operator '=' must be spaced                                   space-infix-ops
   656:23  warning  Operator '=' must be spaced                                   space-infix-ops
   656:28  warning  Operator '=' must be spaced                                   space-infix-ops
   749:19  warning  Identifier 'seen_message_id' is not in camel case             camelcase
   754:38  warning  Identifier 'seen_message_id' is not in camel case             camelcase
   794:31  warning  Operator '=' must be spaced                                   space-infix-ops
   795:10  warning  Operator '=' must be spaced                                   space-infix-ops
   855:21  warning  Identifier 'res_id' is not in camel case                      camelcase
   855:29  warning  Identifier 'res_model' is not in camel case                   camelcase
   860:13  warning  Identifier 'canned_response_ids' is not in camel case         camelcase
   861:24  warning  Operator '=' must be spaced                                   space-infix-ops
   864:18  warning  Operator '=' must be spaced                                   space-infix-ops
   866:13  warning  Identifier 'subtype_id' is not in camel case                  camelcase
  1071:35  warning  Operator '=' must be spaced                                   space-infix-ops
  1126:52  warning  Operator '-' must be spaced                                   space-infix-ops
  1130:68  warning  Operator '+' must be spaced                                   space-infix-ops
  1132:37  warning  Operator '+' must be spaced                                   space-infix-ops
  1152:68  warning  Operator '-' must be spaced                                   space-infix-ops
  1154:37  warning  Operator '-' must be spaced                                   space-infix-ops
  1334:88  warning  A space is required before '}'                                object-curly-spacing
  1584:78  warning  Operator '-' must be spaced                                   space-infix-ops
  1609:49  warning  Operator '=' must be spaced                                   space-infix-ops
  1675:27  warning  Operator '=' must be spaced                                   space-infix-ops
  1676:13  warning  Identifier 'author_id' is not in camel case                   camelcase
  1679:14  warning  Operator '=' must be spaced                                   space-infix-ops
  1681:24  warning  Operator '=' must be spaced                                   space-infix-ops
  1682:13  warning  Identifier 'customer_email_data' is not in camel case         camelcase
  1683:13  warning  Identifier 'customer_email_status' is not in camel case       camelcase
  1685:13  warning  Identifier 'email_from' is not in camel case                  camelcase
  1686:32  warning  Operator '=' must be spaced                                   space-infix-ops
  1688:24  warning  Operator '=' must be spaced                                   space-infix-ops
  1689:24  warning  Operator '=' must be spaced                                   space-infix-ops
  1690:13  warning  Identifier 'is_discussion' is not in camel case               camelcase
  1691:13  warning  Identifier 'is_note' is not in camel case                     camelcase
  1692:13  warning  Identifier 'is_notification' is not in camel case             camelcase
  1693:13  warning  Identifier 'message_type' is not in camel case                camelcase
  1695:13  warning  Identifier 'moderation_status' is not in camel case           camelcase
  1696:13  warning  Identifier 'module_icon' is not in camel case                 camelcase
  1697:35  warning  Operator '=' must be spaced                                   space-infix-ops
  1698:13  warning  Identifier 'record_name' is not in camel case                 camelcase
  1699:13  warning  Identifier 'res_id' is not in camel case                      camelcase
  1700:13  warning  Identifier 'snailmail_error' is not in camel case             camelcase
  1701:13  warning  Identifier 'snailmail_status' is not in camel case            camelcase
  1702:32  warning  Operator '=' must be spaced                                   space-infix-ops
  1704:13  warning  Identifier 'subtype_description' is not in camel case         camelcase
  1705:13  warning  Identifier 'subtype_id' is not in camel case                  camelcase
  1706:13  warning  Identifier 'tracking_value_ids' is not in camel case          camelcase
  1751:22  warning  Identifier 'res_id' is not in camel case                      camelcase
  1752:53  warning  Identifier 'res_id' is not in camel case                      camelcase
  1759:28  warning  Identifier 'author_id' is not in camel case                   camelcase
  1765:34  warning  Identifier 'res_id' is not in camel case                      camelcase
  1766:31  warning  Identifier 'res_id' is not in camel case                      camelcase
  1789:17  warning  Identifier 'record_name' is not in camel case                 camelcase
  1791:37  warning  Identifier 'record_name' is not in camel case                 camelcase
  1809:13  warning  Identifier 'attachment_ids' is not in camel case              camelcase
  1810:32  warning  Identifier 'attachment_ids' is not in camel case              camelcase
  1814:21  warning  Identifier 'is_main' is not in camel case                     camelcase
  1852:13  warning  Identifier 'display_name' is not in camel case                camelcase
  1855:13  warning  Identifier 'im_status' is not in camel case                   camelcase
  1956:33  warning  Operator '=' must be spaced                                   space-infix-ops
  1957:13  warning  Identifier 'channel_type' is not in camel case                camelcase
  1959:13  warning  Identifier 'create_uid' is not in camel case                  camelcase
  1960:13  warning  Identifier 'custom_channel_name' is not in camel case         camelcase
  1961:48  warning  Operator '=' must be spaced                                   space-infix-ops
  1962:13  warning  Identifier 'group_based_subscription' is not in camel case    camelcase
  1964:21  warning  Operator '=' must be spaced                                   space-infix-ops
  1965:24  warning  Operator '=' must be spaced                                   space-infix-ops
  1966:13  warning  Identifier 'is_minimized' is not in camel case                camelcase
  1967:13  warning  Identifier 'is_moderator' is not in camel case                camelcase
  1968:13  warning  Identifier 'mass_mailing' is not in camel case                camelcase
  1969:20  warning  Operator '=' must be spaced                                   space-infix-ops
  1970:13  warning  Identifier 'message_needaction_counter' is not in camel case  camelcase
  1971:13  warning  Identifier 'message_unread_counter' is not in camel case      camelcase
  1975:13  warning  Identifier 'seen_message_id' is not in camel case             camelcase
  1976:13  warning  Identifier 'seen_partners_info' is not in camel case          camelcase
  1983:24  warning  Identifier 'channel_type' is not in camel case                camelcase
  2057:28  warning  Operator '=' must be spaced                                   space-infix-ops
  2289:13  warning  Identifier 'last_message_id' is not in camel case             camelcase
  2290:13  warning  Identifier 'partner_id' is not in camel case                  camelcase
  2332:41  warning  Operator '=' must be spaced                                   space-infix-ops
  2333:13  warning  Identifier 'channel_ids' is not in camel case                 camelcase
  2380:13  warning  Identifier 'last_message_id' is not in camel case             camelcase
  2381:13  warning  Identifier 'partner_id' is not in camel case                  camelcase
  2386:35  warning  Identifier 'partner_id' is not in camel case                  camelcase
  2494:13  warning  Identifier 'channel_type' is not in camel case                camelcase
  2497:13  warning  Identifier 'is_minimized' is not in camel case                camelcase
  2501:13  warning  Identifier 'channel_type' is not in camel case                camelcase
  2506:14  warning  Identifier 'is_minimized' is not in camel case                camelcase
  2549:24  warning  Operator '=' must be spaced                                   space-infix-ops
  2550:24  warning  Operator '=' must be spaced                                   space-infix-ops
  2556:37  warning  Identifier 'message_ids' is not in camel case                 camelcase
  2590:22  warning  Operator '=' must be spaced                                   space-infix-ops
  2594:33  warning  Identifier 'message_ids' is not in camel case                 camelcase
  2691:13  warning  Identifier 'partner_id' is not in camel case                  camelcase
  2748:37  warning  Operator '=' must be spaced                                   space-infix-ops
  2786:13  warning  Identifier 'channel_slots' is not in camel case               camelcase
  2787:21  warning  Operator '=' must be spaced                                   space-infix-ops
  2788:25  warning  Operator '=' must be spaced                                   space-infix-ops
  2789:26  warning  Operator '=' must be spaced                                   space-infix-ops
  2790:40  warning  Operator '=' must be spaced                                   space-infix-ops
  2791:13  warning  Identifier 'menu_id' is not in camel case                     camelcase
  2792:35  warning  Operator '=' must be spaced                                   space-infix-ops
  2793:31  warning  Operator '=' must be spaced                                   space-infix-ops
  2794:37  warning  Operator '=' must be spaced                                   space-infix-ops
  2795:23  warning  Operator '=' must be spaced                                   space-infix-ops
  2796:28  warning  Operator '=' must be spaced                                   space-infix-ops
  2845:28  warning  Operator '=' must be spaced                                   space-infix-ops
  2846:35  warning  Operator '=' must be spaced                                   space-infix-ops
  2847:34  warning  Operator '=' must be spaced                                   space-infix-ops
  2850:28  warning  Identifier 'channel_channel' is not in camel case             camelcase
  2861:28  warning  Identifier 'channel_direct_message' is not in camel case      camelcase
  2872:28  warning  Identifier 'channel_private_group' is not in camel case       camelcase
  2917:13  warning  Identifier 'is_moderator' is not in camel case                camelcase
  2918:13  warning  Identifier 'moderation_counter' is not in camel case          camelcase
  2919:13  warning  Identifier 'needaction_inbox_counter' is not in camel case    camelcase
  2920:13  warning  Identifier 'starred_counter' is not in camel case             camelcase
  2923:51  warning  Identifier 'needaction_inbox_counter' is not in camel case    camelcase
  2924:53  warning  Identifier 'starred_counter' is not in camel case             camelcase
  2925:13  warning  Identifier 'is_moderator' is not in camel case                camelcase
  3313:23  warning  Operator '=' must be spaced                                   space-infix-ops
  3313:28  warning  Operator '=' must be spaced                                   space-infix-ops
  3358:23  warning  Operator '=' must be spaced                                   space-infix-ops
  3358:28  warning  Operator '=' must be spaced                                   space-infix-ops
  3400:14  warning  Operator '*' must be spaced                                   space-infix-ops
  3413:15  warning  Operator '-' must be spaced                                   space-infix-ops
  3544:18  warning  Operator '=' must be spaced                                   space-infix-ops
  3545:17  warning  Operator '=' must be spaced                                   space-infix-ops
  3546:10  warning  Operator '=' must be spaced                                   space-infix-ops
  3930:23  warning  Operator '=' must be spaced                                   space-infix-ops
  3931:9   warning  Identifier 'author_id' is not in camel case                   camelcase
  3934:10  warning  Operator '=' must be spaced                                   space-infix-ops
  3936:20  warning  Operator '=' must be spaced                                   space-infix-ops
  3937:9   warning  Identifier 'customer_email_data' is not in camel case         camelcase
  3938:9   warning  Identifier 'customer_email_status' is not in camel case       camelcase
  3940:9   warning  Identifier 'email_from' is not in camel case                  camelcase
  3941:28  warning  Operator '=' must be spaced                                   space-infix-ops
  3942:20  warning  Operator '=' must be spaced                                   space-infix-ops
  3943:9   warning  Identifier 'is_discussion' is not in camel case               camelcase
  3944:9   warning  Identifier 'is_note' is not in camel case                     camelcase
  3945:9   warning  Identifier 'is_notification' is not in camel case             camelcase
  3946:9   warning  Identifier 'message_type' is not in camel case                camelcase
  3948:9   warning  Identifier 'moderation_status' is not in camel case           camelcase
  3949:9   warning  Identifier 'module_icon' is not in camel case                 camelcase
  3950:31  warning  Operator '=' must be spaced                                   space-infix-ops
  3951:9   warning  Identifier 'record_name' is not in camel case                 camelcase
  3952:9   warning  Identifier 'res_id' is not in camel case                      camelcase
  3953:9   warning  Identifier 'snailmail_error' is not in camel case             camelcase
  3954:9   warning  Identifier 'snailmail_status' is not in camel case            camelcase
  3955:28  warning  Operator '=' must be spaced                                   space-infix-ops
  3957:9   warning  Identifier 'subtype_description' is not in camel case         camelcase
  3958:9   warning  Identifier 'subtype_id' is not in camel case                  camelcase
  3959:9   warning  Identifier 'tracking_value_ids' is not in camel case          camelcase
  4002:22  warning  Identifier 'res_id' is not in camel case                      camelcase
  4003:53  warning  Identifier 'res_id' is not in camel case                      camelcase
  4014:31  warning  Identifier 'author_id' is not in camel case                   camelcase
  4021:34  warning  Identifier 'res_id' is not in camel case                      camelcase
  4022:31  warning  Identifier 'res_id' is not in camel case                      camelcase
  4036:42  warning  Identifier 'attachment_ids' is not in camel case              camelcase
  4095:17  warning  Identifier 'record_name' is not in camel case                 camelcase
  4097:44  warning  Identifier 'record_name' is not in camel case                 camelcase

✖ 169 problems (0 errors, 169 warnings)
  0 errors and 70 warnings potentially fixable with the `--fix` option.

Reply:
For camelcase you can disable the rule.

For space-infix-ops we actually need to fix our code.

Reply:
Mmh disagree for camelcase as everywhere we use the convention of camel cases except some exception... Just try to create a component with a property "foo_bar" and you'll unleash the Alex Kraken and he will shout at you : "fooBar not foo_bar !"  !

Reply:
I meant disable the rule in the linter, but still follow it. It's pretty easy when creating a variable to keep it in mind, and also easy to notice manually at review when it is wrong.